libs/sdk-backend-spi/src/workspace/execution/index.ts

```diff
@@ -315,6 +315,11 @@ export interface IDataView {
      */
     readonly result: IExecutionResult;
 
+    /**
+     * Result warnings.
+     */
```

Comment:
please add more contractual info here. something like. 

Backend MAY pass warnings to indicate that the result is different compared to what the caller may expect. For instance the caller executes a definition with particular filter; backend determines that the filter does not have any impact on the computation; backend runs the execution and communicates via warning the the filter was useless.